android/src/com/mapswithme/maps/MapFragment.java

```diff
@@ -90,8 +90,6 @@ private void setupWidgets(int width, int height)
                       UiUtils.dimen(context, R.dimen.margin_base),
                       ANCHOR_LEFT_TOP);
 
-    mCurrentCompassOffsetX = 0;
-    mCurrentCompassOffsetY = UiUtils.getCompassYOffset(requireContext());
```

Comment:
In this case I don't understand the logic.
- mCurrentCompassOffsetX and mCurrentCompassOffsetY are assigned in setupCompass only
- setupCompass also reads this variables first

mCurrentCompassOffsetX,  mCurrentCompassOffsetY will be uninitialized?

Reply:
But we should ensure that this call was happened before, somehow ..